doc/clang.txt

```diff
@@ -86,6 +86,11 @@ g:clang_cpp_completeopt                 *g:clang_cpp_completeopt*
         " Have preview window for C++ as its complex prototypes
         let g:clang_cpp_completeopt = 'longest,menuone,preview'
 <
+g:clang_debug                           *g:clang_debug*
+        Turns on debug mode with value being verbosity
+        Default: >
+            let g:clang_debug = 1
```

Comment:
Cheers. Fixed the default and added information about the values.

Am I right in saying the max input value for s:PDebug so far is 3?